pkgs/by-name/li/litecli/package.nix

```diff
@@ -21,8 +24,9 @@ python3Packages.buildPythonApplication rec {
     sqlparse
   ];
 
-  nativeCheckInputs = with python3Packages; [
+  build-system = with python3Packages; [
```

Comment:
```suggestion
  nativeCheckInputs = with python3Packages; [
```

Reply:
`setuptools` should be in `build-system`, while the rest is still `nativeCheckInputs`